docs/commands.adoc

```diff
@@ -669,6 +669,7 @@ Requires a vector to be used as the source of the search which can be supplied i
 For example imagine we have the following document stored in our cluster:
 
 [options="nowrap"]
+```
```

Comment:
Seems unrelated to this change?

Reply:
It's fixing a bug, I think it accidentally got deleted in one of the other recipe PRs